gnosis/cowswap
# [ABA 🎵 Impact] - Fallback message and Loader

# Summary

Closes #1908

Description && screenshots in #1908 

TODO: there are some tweaks to make:
1. ~better hiding/showing of message based on price impact - idea will be to (in another PR) make a global flag for ABA impact loading/not existing and wait for that to return before showing/hiding message~ done using fallback price impact error checking
2. ~a loader potentiatlly?~ done loader in commit

### LOADER screenshot
![Screenshot 2021-11-29 at 7 50 32 PM](https://user-images.githubusercontent.com/21335563/143925742-f7235270-ecdc-40ce-af21-58854527ad7d.png)

  # To Test
1. any chain
2. any tokens
3. enter amounts
4a. see loader in price impact spot
4. wait for message to appear (if selected token pair has no impact)
5. switch to other token with known impact
6. message warning should disappear
